pkg/operator/operator.go

```diff
@@ -387,8 +419,19 @@ func (o *Operator) Config(key string) (*manifests.Config, error) {
 		return nil, err
 	}
 
+	// Only use User Workload Monitoring ConfigMap from user ns and populate if
+	// its enabled by admin via Cluster Monitoring ConfigMap.  The above
+	// loadConfig() already initializes the structs with nil values for
+	// UserWorkloadConfiguration struct.
+	if *c.ClusterMonitoringConfiguration.UserWorkloadEnabled {
+		c.UserWorkloadConfiguration, err = o.loadUserWorkloadConfig()
+		if err != nil {
+
```

Comment:
i believe this is WIP

Reply:
or isn't it? :-) do we really want to swallow the error?

Reply:
nope, just me forgetting to return error :( fixed